system/modules/core/forms/FormTextArea.php

```diff
@@ -144,7 +144,7 @@ public function __get($strKey)
 				break;
 
 			case 'value':
-				return specialchars(str_replace('\n', "\n", $this->varValue));
+				return str_replace('\n', "\n", $this->varValue);
```

Comment:
I read it. But still, the behavior was since version 3.3.0 that `->value` of the textarea returned the encoded value. Every use of it that relies on that fact may break in an unsecure way.